services/frontend-v3/src/components/layouts/profileLayout/ProfileLayoutView.jsx

```diff
@@ -330,19 +341,37 @@ const ProfileLayoutView = ({ data }) => {
           </Row>
         )}
 
-        {visibleCards.talentManagement && visibleCards.careerInterests ? (
+        {(visibleCards.talentManagement ||
+          (visibleCards.exFeeder && data.exFeeder)) &&
```

Comment:
why are we checking both visibleCards.exFeeder && data.exFeeder

Reply:
Currently on development it is not possible to show you are not an ex-feeder. If you don't check the box no message will show on your profile. I assumed this we would want to keep this because I don't think that is something someone would ever want to publicly advertise they aren't an ex-feeder